vertx-web/src/main/java/io/vertx/ext/web/handler/impl/CorsHandlerImpl.java

```diff
@@ -188,7 +195,9 @@ public void handle(RoutingContext context) {
         if (maxAgeSeconds != null) {
           response.putHeader(ACCESS_CONTROL_MAX_AGE, maxAgeSeconds);
         }
-
+        if (request.headers().contains(ACCESS_CONTROL_REQUEST_PRIVATE_NETWORK)) {
```

Comment:
do we always want to send this header?
Should we deal with it as a tri-state? if config is null, do nothing (old behavior), either true or false, then apply it?

PS: I didn't read the spec regarding the header so I may be saying something totally wrong here!

Reply:
yes you are right, i did the same for the netty handler as well

Reply:
i did not find any hint in the rfc if it is even allowed to return false - if it's similar to the credentials one, it should only return true or omit the whole header